beamer/agent.py

```diff
@@ -51,6 +51,10 @@ def __init__(self, config: Config):
         request_manager = l2a_contracts["RequestManager"]
         fill_manager = l2b_contracts["FillManager"]
 
+        max_validity_period = request_manager.functions.MAX_VALIDITY_PERIOD().call()
+
+        if config.unsafe_fill_time >= max_validity_period:
+            raise RuntimeError(f"Unsafe fill time must not be bigger than {max_validity_period}")
```

Comment:
Nitpick: slight better would be "must be less than" instead of "must not be bigger than" because we're also disallowing them being equal.